styles.css

```diff
@@ -216,15 +216,18 @@ ul {
 }
 
 .bullet {
-  width: 10px;
-  height: 10px;
-  background: #ccc;
-  border-radius: 50%;
+  /* width: 10px;
+  height: 10px; */
```

Comment:
[OPTIONAL] Please don't leave any uncommented code in your codebase. So you can keep your codebase neat and professional 👨‍💼